src/content/ja/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -546,17 +547,17 @@ Key Point: このコードラボでは、サービスワーカーのライフサ
 
 リソースを事前に把握することは、ユーザーがデスクトップまたはモバイルアプリをインストールしたときに起こることと似た概念です。アプリの実行に必要な主要なリソースは、ネットワーク接続があるかどうかにかかわらず後でロードできるように、インストールされるか、デバイスにキャッシュされます。
 
-私たちのアプリでは、サービスワーカーがインストールされたときにすべての静的リソースを事前キャッシュして、アプリの実行に必要なすべてのものがユーザーのデバイスに保存されるようにします。我々のアプリが速く速くロードすることを確実にするために、我々は[cache-first](/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)戦略を使います。ネットワークにアクセスしてリソースを取得する代わりに、それらはローカルキャッシュから取得されます。それが利用できない場合にのみ、ネットワークから取得しようとします。
+私たちのアプリでは、Service Workerがインストールされたときにすべての静的リソースを事前キャッシュして、アプリの実行に必要なすべてのものがユーザーのデバイスに保存されるようにします。我々のアプリが速く速くロードすることを確実にするために、[cache-first](/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)戦略を使います。ネットワークにアクセスしてリソースを取得する代わりに、それらはローカルキャッシュから取得されます。それが利用できない場合にのみ、ネットワークから取得しようとします。
 
 ![44860840e2090bd8.png](img/44860840e2090bd8.png)
 
 ローカルキャッシュから取得することで、ネットワークの変動がなくなります。ユーザーがどのような種類のネットワーク（WiFi、5G、3G、さらには2Gまで）に接続していても、実行する必要がある主要なリソースはほとんどすぐに利用できます。
 
-Caution: このサンプルでは、静的リソースは[`cache-first`](/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)戦略を使用して[`cache-first`](/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)されているため、ネットワークに[`cache-first`](/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)ことなくキャッシュされたコンテンツのコピーが返されます。 `cache-first`戦略は実装が簡単ですが、将来的に課題を引き起こす可能性があります。
+警告: このサンプルでは、静的リソースは[`cache-first`](/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)戦略を使用して提供されます。これによりネットワークに関係なくキャッシュされたコンテンツのコピーが返されることになります。`cache-first`戦略は実装が簡単ですが、将来的に課題を引き起こす可能性があります。
```

Comment:
Please leave `Caution:` untranslated, it's a special keyword that does some extra formatting. The same goes for Note:, Warning:, Success:, Key Point:, Key Term:, Objective:

Would you double check that these aren't translated throughout?

Reply:
Done